src/content/docs/fr/guides/integrations-guide/sitemap.mdx

```diff
@@ -82,35 +82,70 @@ export default defineConfig({
 
 ## Utilisation
 
-`@astrojs/sitemap` nécessite une URL de déploiement / site pour la génération. Ajoutez l'URL de votre site dans votre `astro.config.*` en utilisant la propriété `site`. Celle-ci doit commencer par `http:` ou `https:`.
+`@astrojs/sitemap` a besoin de connaître l'URL déployée de votre site pour générer un sitemap.
 
-```js title="astro.config.mjs" "'https://stargazers.club'"
+Ajoutez l'URL de votre site comme option [`site`](/fr/reference/configuration-reference/#site) dans `astro.config.mjs`. Elle doit commencer par `http://` ou `https://`.
+
+```js title="astro.config.mjs" {5}
 import { defineConfig } from 'astro/config';
 import sitemap from '@astrojs/sitemap';
 
 export default defineConfig({
-  // ...
   site: 'https://stargazers.club',
   integrations: [sitemap()],
+  // ...
 });
 ```
 
-Notez que contrairement aux autres options de configuration, `site` est défini dans l'objet racine `defineConfig`, plutôt que dans l'appel à `sitemap()`.
+Avec l'intégration du sitemap configurée, les fichiers `sitemap-index.xml` et `sitemap-0.xml` seront ajoutés à votre répertoire de sortie lors de la construction de votre site.
 
-Maintenant, [construisez votre site pour la production](/fr/reference/cli-reference/#astro-build) via la commande `astro build`. Vous trouverez `sitemap-index.xml` et `sitemap-0.xml` dans le dossier `dist/` (ou dans votre [répertoire de sortie](/fr/reference/configuration-reference/#outdir) si vous l'avez défini).
+`sitemap-index.xml` renvoie à tous les fichiers sitemap numérotés.
+`sitemap-0.xml` liste les pages de votre site.
+Pour les sites de très grande taille, il peut y avoir des fichiers numérotés supplémentaires comme `sitemap-1.xml` et `sitemap-2.xml`.
 
-:::caution
-Si vous oubliez d'ajouter un `site`, vous recevrez un avertissement amical lors de la construction, et le fichier `sitemap-index.xml` ne sera pas généré.
-:::
+<details>
+<summary>Exemple de fichiers générés pour un site web de deux pages</summary>
+
+```xml title="sitemap-index.xml"
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <sitemap>
+    <loc>https://stargazers.club/sitemap-0.xml</loc>
+  </sitemap>
+</sitemapindex>
+```
+
+```xml title="sitemap-0.xml"
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
+  <url>
+    <loc>https://stargazers.club/</loc>
+  </url>
+  <url>
+    <loc>https://stargazers.club/second-page/</loc>
+  </url>
+</urlset>
+```
+</details>
+
+### Découverte du plan du site
+
+Vous pouvez faciliter la recherche de votre sitemap par les robots d'indexation en insérant des liens dans le fichier `<head>` et `robots.txt` de votre site.
```

Comment:
A small reword because `<head>` is not a file:
```suggestion
Vous pouvez faciliter la recherche de votre sitemap par les robots d'indexation en insérant des liens dans `<head>` et dans le fichier `robots.txt` de votre site.
```

Alternatively, I would also reword the beginning but it is not mandatory:
```suggestion
Vous pouvez permettre aux robots d'indexation de trouver plus facilement votre sitemap en insérant des liens dans `<head>` et dans le fichier `robots.txt` de votre site.
```
So you can choose any suggestion.